src/Microsoft.ML.Data/Transforms/RowToRowTransformerBase.cs

```diff
@@ -23,7 +23,8 @@ protected RowToRowTransformerBase(IHost host)
             Host = host;
         }
 
-        public abstract void Save(ModelSaveContext ctx);
+        void ICanSaveModel.Save(ModelSaveContext ctx) => SaveModel(ctx);
+        private protected abstract void SaveModel(ModelSaveContext ctx);
```

Comment:
part of me want you to search for all 
```
void ICanSaveModel.Save(ModelSaveContext ctx) => SaveModel(ctx);
private protected abstract void SaveModel(ModelSaveContext ctx);
```
and replace them to
```
void ICanSaveModel.Save(ModelSaveContext ctx) => SaveModel(ctx);

private protected abstract void SaveModel(ModelSaveContext ctx);
```
but it's up to you

Reply:
I can replace that sure!

---
In reply to: [255247150](https://github.com/dotnet/machinelearning/pull/2431#discussion_r255247150) [](ancestors = 255247150)

Reply:
Only 4 occurrences :) 

---
In reply to: [255276034](https://github.com/dotnet/machinelearning/pull/2431#discussion_r255276034) [](ancestors = 255276034,255247150)